src/index/formatFileStructure/buildGraph.ts

```diff
@@ -1,55 +1,54 @@
 import path from "path";
 import { findEdges } from "./shared/findEdges";
-import { addEdge } from "./buildGraph/addEdge";
+import { addEdgeToGraph } from "./buildGraph/addEdge";
 import { Graph, OldGraph } from "./shared/Graph";
 import { findSharedParent } from "./shared/findSharedParent";
 import { customResolve } from "./shared/customResolve";
+import OS from "os";
```

Comment:
What does this do? It seems like its not used anywhere.

Reply:
Nothing now. I was figuring something out. I'll get rid of this.